backend/auto_operation/Components.py

```diff
@@ -264,6 +266,7 @@ def __init__(
         self.mileage = initialPosition
         self.prevMileage = initialPosition
         self.pidParam = pidParam
+        self.stopPoint: StopPoint | None = None  # 停止点
```

Comment:
```suggestion
        self.stopPoint: Optional[StopPoint] = None  # 停止点
```
が同値です。Optionalを使っている部分の方が多かったイメージがあるので揃えちゃいましょう